Crypto landing page copy/index.html

```diff
@@ -0,0 +1,482 @@
+                    </ul>
+                    <div class="header-button">
+                        <a href="#">Sign in</a>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </header>
+    <section class="section1">
+        <div class="container">
+            <div class="container-fluid">
+                <div class="elips">
+                    <div class="elips-img">
+                        <img src="img/ellipse.png" alt="ellipse">
+                    </div>
+                    <h1>Live And<br>
+                        on-demand
+                        trading
+                    </h1>
+                    <p>
+                        Lorem ipsum dolor sit amet. Qui consequatur sint 33 volupta officia et sint laboriosam sed
+                        ipsa
+                        sint ut voluptatum labore et possimus voluptas.
+                    </p>
+                    <div class="section1-bottom">
+                        <a href="#">Explore Now</a>
+                        <img src="img/page.png" alt="page navigator">
+                    </div>
+                </div>
+                <div class="section1-img">
+                    <img src="img/iphone.png" alt="Photo of the iPhone 12">
+                </div>
+            </div>
+        </div>
+        </div>
+    </section>
+    <section class="section2">
+        <div class="container">
+            <div class="section2-wrapper">
+                <ul>
+                    <li>
+                        <a href="#"><img src="img/waverio.png" alt="logo waverio"></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="img/logipsum.png" alt="logo logipsum"></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="img/alterbone.png" alt="logo Alterbone"></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="img/ridoria.png" alt="logo Ridoria"></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="img/tinygone.png" alt="logo Tinygone"></a>
+                    </li>
+                </ul>
+            </div>
+        </div>
+    </section>
+    <section class="section3">
+        <div class="container">
+            <div class="section3-wrapper">
+                <div class="section3-post">
+                    <h2>
+                        Automated portfolio tracking
+                    </h2>
+                    <p>
+                        Lorem ipsum dolor sit amet. Qui consequatur sint 33 voluptatem officia et sint laboriosam sed
+                        ipsa sint ut voluptatum labore et possimus voluptas.
+                    </p>
+                </div>
+                <div class="section3-item-container">
+                    <div class="section3-item">
+                        <h1>
+                            01
```

Comment:
and make it as `:before` element. You reset the counter on the .section3-items, and for .section3-item:before { } will be a counter number. So if more items are in the feature, the number will be set automatically.
For example - https://doka.guide/css/counter-counters/